### DISCUSSION TITLE
use of librebooking for 2 types of people ?

### DISCUSSION THREAD

**Discussion by Betus971** [LibreBooking/librebooking#434] [Q&A] [answered]
Hello for librebooking I would like to set up 2 types of reservation
1. room
2. hairdresser
I have to install 2 librebooking with 2 different links or just 1 with 2 different links when my user clicks on the hairdresser tab he can book a session at the hairdresser and when he clicks for the rooms he can book the rooms is it possible?

**Comment by apezzi75** [comment]
You can use only one librebooking, but two type of Resource, the type Room and the type Hairdresser

**Comment by effgarces** [comment, marked as answer]
For Librebooking the room, the hairdresser are simply resources, so no need for 2 installs, it's 1 install with both resources, what will differ is  how you configure the rules on both resources, it might even be advisable to create different schedules/calendars for each.

**Reply by Betus971** [reply]
thank you so it is possible? when the user wants to book
it will be displayed
Conference room1
conference room2
hairdresser

**Comment by apezzi75** [comment]
ConferenceRoom1 and ConferenceRoom2 are the default resource. If you don't need, you can delete them.
Then you can create a new Type of resource named Room, and create 2 new resource named Room1 e Room2 of this Type. 


Il giorno 10 ott 2024, 12:07, alle ore 12:07, Deep in misery ***@***.***> ha scritto:

**Reply by Betus971** [reply]
thanks a lot

**Comment by treborin** [comment]
Perdona, @Richar1481 ,
**No deberías** duplicar este asunto en distintos sitios.
Ya te he contestado en otro hilo: https://github.com/LibreBooking/app/discussions/447
